### ISSUE TITLE
Running on WildTrack Dataset

### ISSUE THREAD

**Issue by TTTREE** [pierrebaque/EM#43]
Hi!
This is a nice project!
I am trying to run this code on WildTrack Dataset to get the detection results. And I don't know if I need to modify the config.py or other files. So would you please tell me how to run this DeepOcclusion on WildTrack Dataset.
More detail better!
Thanks a lot !!!!!

**Comment by FoFb8alted** [comment]
Did you manage to get the results for wildtracks? Also, did you manage to get the code to run?

**Comment by TTTREE** [comment]
> 
> 
> Did you manage to get the results for wildtracks? Also, did you manage to get the code to run?

No, I even cannot run the code because there is something wrong with theano environment. So I give up.

**Comment by leconghieu99** [comment]
@FoFb8alted can you please send me the .zip files. Looks like the link is broken. Email: Hieus.lecong@gmai.com

**Comment by leconghieu99** [comment]
@TTTREE can you please send me the .zip files. Looks like the link is broken. Email: Hieus.lecong@gmai.com
